central/complianceoperator/v2/report/manager/utils/results_aggregator.go

```diff
@@ -0,0 +1,206 @@
+package utils
```

Comment:
I am not a fan of `utils` packages. I see a chance to name it `results`. Then we could use it like this: `results. NewAggregator `.

Reply:
The `utils` package contains other things like errors, helper functions for the snapshot store, and (in a follow-up PRs) more helper functions. I don't think that the name `results` works for the other things in this package. I'll open a follow-up to separate the different parts of `utils` into their own packages but for now I'll like to keep it as it is.